docs/Database_Schema.md

```diff
@@ -448,3 +448,22 @@ Create TABLE IF NOT EXISTS "marketcap" (
     PRIMARY KEY ("last_updated_at")
 );
 ```
+
+## 16. Table **tx_pool**
+
+### _Schema_
+
+| Column            | Data Type | PK | Not NULL | Default  |Description|
+|:----------------- |:--------- |:--:|:--------:| -------- | --------- |
+| hash              | TEXT      | Y  | Y        |          | The hash of transaction|
```

Comment:
```suggestion
| hash              | TEXT      | Y  | Y        |          | The text-encoded hash of transaction |
```

Reply:
I changed the name to be the same as Agora, please review it again.